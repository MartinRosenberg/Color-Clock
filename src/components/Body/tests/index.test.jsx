import { shallow } from 'enzyme'
import React from 'react'

import { Body } from '..'

describe('Body component', function () {
  test('should render correctly', () => {
    const wrapper = shallow(<Body color={'#123456'}/>)
    expect(wrapper).toMatchSnapshot()
  })
})
