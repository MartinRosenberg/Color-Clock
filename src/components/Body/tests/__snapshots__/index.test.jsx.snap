// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Body component should render correctly 1`] = `
<HelmetWrapper
  defer={true}
  encodeSpecialCharacters={true}
>
  <body
    style="background: #123456"
  />
</HelmetWrapper>
`;
